OpenDreamRuntime/Procs/Native/DreamProcNativeRoot.cs

```diff
@@ -2489,5 +2489,31 @@ public static DreamValue NativeProc_viewers(DreamObject instance, DreamObject us
             connection.WinSet(winsetControlId, winsetParams);
             return DreamValue.Null;
         }
+
+        [DreamProc("winclone")]
+        [DreamProcParameter("player", Type = DreamValueType.DreamObject)]
+        [DreamProcParameter("window_name", Type = DreamValueType.String)]
+        [DreamProcParameter("clone_name", Type = DreamValueType.String)]
+        public static DreamValue NativeProc_winclone(DreamObject instance, DreamObject usr, DreamProcArguments arguments) {
```

Comment:
This needs to be moved above `winexists` to be kept in alphabetic order. There also needs to be a `SetGlobalNativeProc()` call in `DreamProcNative.cs` for this to be used.